### ISSUE TITLE
Still getting CORS errors

### ISSUE THREAD

**Issue by bibhas2** [hoppscotch/hoppscotch-extension#50]
I have installed the extension in Chrome. It's not clear exactly how to use this. This is what I have done:

I have added my local server to the list.

![image](https://user-images.githubusercontent.com/1730356/117690158-8c278180-b188-11eb-80f8-4eee888d1151.png)

I have added the local server to the active origins.

![image](https://user-images.githubusercontent.com/1730356/117690246-a2354200-b188-11eb-8d51-6415098461c4.png)

I am still getting CORS error when I send a request to localhost:8080.

**Comment by liyasthomas** [comment]
Can you make sure that you've turned on "Extension mode" from Hoppscotch > [Settings](https://hoppscotch.io/settings) > Extensions section "Use browser extension to send requests".

Let me know if it solves your issue.

**Comment by AndrewBastin** [comment]
Is `http://localhost:8080/` the server you are trying to connect to or a locally hosted Hoppscotch instance ?

The "Active Origins" list is for the extension to decide which all pages to hook to. The extension hooks some extra functions into the JS context of the pages defined in the list.

**Comment by bibhas2** [comment]
> Can you make sure that you've turned on "Extension mode" from Hoppscotch > [Settings](https://hoppscotch.io/settings) > Extensions section "Use browser extension to send requests".
> 

This was enabled by default already.

**Comment by bibhas2** [comment]
I managed to get things working by giving site access for all sites. I don't know what that means. But it does not appear a very secure thing to do. I am running into significant mental friction with this tool. I have used Postman and other tools including something I had developed. May be better documentation will help. It's very common for developers to be able to test non-CORS enabled services.

![image](https://user-images.githubusercontent.com/1730356/117723780-d290d680-b1b0-11eb-8144-eeb48cf95f39.png)
